src/config.ts

```diff
@@ -33,9 +34,17 @@ function _trimAliasName(name: string): string {
 
 async function readEdnWorkspaceConfig(uri?: vscode.Uri) {
   try {
-    const data = await vscode.workspace.fs.readFile(
-      uri ?? vscode.Uri.file(state.resolvePath('.calva/config.edn'))
-    );
+    let resolvedUri: vscode.Uri;
+    const configPath = state.resolvePath('.calva/config.edn');
+
+    if (isDefined(uri)) {
+      resolvedUri = uri;
+    } else if (isDefined(configPath)) {
+      resolvedUri = vscode.Uri.file(configPath);
+    } else {
+      throw new Error('Expected a uri to be passed in or a config to exist at .calva/config.edn');
+    }
+    const data = await vscode.workspace.fs.readFile(resolvedUri);
```

Comment:
Fixing the issue where `state.resolvePath` can return undefined and make sure we have a better error message when that's the case.